doc/OperationalMonitoring/Architecture/arc-opmond_x-road_operational_monitoring_daemon_architecture_Y-1096-1.md

```diff
@@ -161,7 +161,7 @@ Figure 2 shows the deployment diagram.
 **Figure 2. Operational monitoring daemon deployment**
 
 
-<a name="AppendixA"/></a>
+<a name="AppendixA"></a>
```

Comment:
This should likely use the same `<a />` style rather than `<a></a>` style, similar to the other documents e.g the one in ig-xlb_x-road_external_load_balancer_installation_guide.md.